database/tables/batched_migration_jobs.pg

```diff
@@ -0,0 +1,20 @@
+columns
+    attempts: integer default 0
+    batched_migration_id: bigint not null
+    created_at: timestamp with time zone default CURRENT_TIMESTAMP
+    data: jsonb
+    finished_at: timestamp with time zone
+    id: bigint not null default nextval('batched_migration_jobs_id_seq'::regclass)
+    max_value: bigint not null
+    min_value: bigint not null
+    started_at: timestamp with time zone
+    status: enum_batched_migration_job_status default 'pending'::enum_batched_migration_job_status
+    updated_at: timestamp with time zone default CURRENT_TIMESTAMP
+
+indexes
+    batched_migration_jobs_pkey: PRIMARY KEY (id) USING btree (id)
+    batched_migration_jobs_batched_migration_id_max_value_idx: USING btree (batched_migration_id, max_value)
+    batched_migration_jobs_batched_migration_id_status_idx: USING btree (batched_migration_id, status)
```

Comment:
This could be used to help make sure two servers don't attempt to do a job on the same range at the same time. I suppose this would be a rare race condition, and I know the operation must be idempotent so it won't lead to inconsistency issues, its just nice to not duplicate work.

Reply:
Ok, I'm fine without them.